src/App.vue

```diff
@@ -1,11 +1,14 @@
 <template>
   <div id="header">
+    <div>
+      <LocaleSwitcher id="locale" class="hidden"></LocaleSwitcher>
```

Comment:
```suggestion
      <LocaleSwitcher id="locale" v-show="false"></LocaleSwitcher>
```

Reply:
Nice :+1:

Reply:
This removes the element - does not make it invisible.